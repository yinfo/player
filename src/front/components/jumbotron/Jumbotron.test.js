// @flow

import React from 'react';
import { shallow } from 'enzyme';
import Jumbotron from './Jumbotron';

describe('Jumbotron component', () => {
  it('renders as expected', () => {
    const component = shallow(
      <Jumbotron>
        <p>a child element</p>
      </Jumbotron>,
    );
    expect(component).toMatchSnapshot();
  });
});
