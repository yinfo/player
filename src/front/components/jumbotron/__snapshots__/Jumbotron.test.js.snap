// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbotron component renders as expected 1`] = `ShallowWrapper {}`;
